docker/dev/django/Dockerfile

```diff
@@ -8,4 +8,8 @@ WORKDIR /code
 ADD requirements/* /code/
 RUN pip install -r dev.txt
 
+RUN pip install --upgrade pip
```

Comment:
Do we need this?

Reply:
We can upgrade the pip version so that we don't need to run this command everytime